2025/build-tooling/slides.md

```diff
@@ -16,61 +16,59 @@ is: feedback
 
 ---
 
-# Audience
+# Are you looking to...
 
-- Developers looking to add build step to their apps
-- Developers looking to learn more about the capabilities of the build tools
-- Developers interested in how Esri's teams are leveraging modern build tools
+- Add a build step to your app?
+- Learn more about build tool capabilities?
```

Comment:
Thanks for the refactor - I can see now that the order of items can be changed to be more logical:

```suggestion
- Learn about build tool capabilities?
- Add a build step to your app?
```

eg, people should first learn what are build tools and why, before learning how to add build tools to their apps